rdr_service/resource/generators/participant.py

```diff
@@ -594,12 +594,12 @@ def _prep_participant_profile(self, p_id, ro_session):
                     ParticipantEhrReceipt.firstSeen,
                     ParticipantEhrReceipt.fileTimestamp
                 )
-            ce_hpo_id = get_ce_mediated_hpo_id()
-            if ce_hpo_id is not None:
+            ce_hpo_id_list = get_ce_mediated_hpo_id_list()
+            if ce_hpo_id_list is not None:
```

Comment:
FYI: The PDR team is discussing whether we need to allow CE mediated EHR entries to be included in this data build.  It would have to add a new key/value pair to the `ehr_receipts` record/data dict  to indicate  it was a mediated EHR file or not.

We'll plan that as a separate PR, if needed.  We may decide to handle it in the new pipeline and not make any more changes to this generator code.